conkit/plot/contactplotplus.py

```diff
@@ -0,0 +1,267 @@
+            self.canvas.add_tools(
+                HoverTool(renderers=[self.sspred_h], tooltips=[('Position', '@x'), ('Residue Name', '@aa_name'),
+                                                               ('Properties', '@aa_properties')]))
+            self.sspred_c = self.canvas.circle(x='x', y='y', color='blue', size=3, source=source3)
+            self.canvas.add_tools(
+                HoverTool(renderers=[self.sspred_c], tooltips=[('Position', '@x'), ('Residue Name', '@aa_name'),
+                                                               ('Properties', '@aa_properties')]))
+            self.sspred_e = self.canvas.circle(x='x', y='y', color='pink', size=3, source=source4)
+            self.canvas.add_tools(
+                HoverTool(renderers=[self.sspred_e], tooltips=[('Position', '@x'), ('Residue Name', '@aa_name'),
+                                                               ('Properties', '@aa_properties')]))
+        else:
+            raise ValueError('ss prediction is not the same length as sequence')
+
+    def add_dispred_layer(self, dispred):
+
+        self.dispred = dispred
+
+        if len(dispred) == len(self.conpred.sequence.seq):
+
+            x_d = [residue.id for residue in self.dispred.search(0.5)]
+            x_nd = [residue.id for residue in self.dispred if residue.id not in x_d]
+            y_d = [residue.id for residue in self.dispred.search(0.5)]
+            y_nd = [residue.id for residue in self.dispred if residue.id not in y_d]
+            y_d = [int(y) - 3 for y in y_d]
+            y_nd = [int(y) - 3 for y in y_nd]
+
+            self.dispred_o = self.canvas.circle(x_nd, y_nd, color='grey', size=5)
+            self.dispred_dis = self.canvas.circle(x_d, y_d, size=5, color='lime')
+        else:
+            raise ValueError('disorder prediction is not the same length as sequence')
+
+    def add_conservpred_layer(self, conservpred):
+        self.conservpred = conservpred
+
+        if len(conservpred) == len(self.conpred.sequence.seq):
+            consurf_values = list(range(1, 10))
+
+            x_dct = {}
+            y_dct = {}
+
+            for score in consurf_values:
+                x_dct[score] = [residue.id for residue in self.conservpred.search(score)]
+                y_dct[score] = [int(x) + 3 for x in x_dct[score]]
+
+            # for x in self.spectrum:
+            #     self.canvas.triangle(x_dct[x], y_dct[x], size=5, color=self.spectrum[x])
+
+            # couldn't get this into a single line loop????
+            for x in self.spectrum.keys():
+                self.canvas.triangle(x_dct[x], y_dct[x], size=5, color=self.spectrum[x])
+
+
+        else:
+            raise ValueError('conservation prediction is not the same length as sequence')
+
+    def plot(self, fname):
+        # print(self.layers)
+        self.canvas.add_layout(self.legend, 'right')
+
+        output_file(fname)
+        show(self.canvas)
```

Comment:
why show this? there should be function called ``show``. ``plot`` plots, that's it